README.md

```diff
@@ -427,3 +459,94 @@ You are busy on this interval!
+
+What you probably want to do next:
+
+Run /isfree again with different datetime
+```
+
+### 4.8 See All Your Upcoming Events
+
+You can view your upcoming events through the **/getupcomingevents** command. The bot will respond in the following manner:
+
+```
+Please enter how many upcoming events are you looking for!
+
+Press!
+```
+It will then display a markup keyboard displaying a number from 1 to 7. You can choose to display upto 7 upcoming events. 
+In the case where you clicked **7**, the bot will respond in this following manner:
+```
+Getting 7 upcoming event(s) for you
+Here they are!
+```
+And it will display the first seven of your upcoming events (including your courses).
+It displays the **name of the event, date of the event and time interval of the event**.
+
+### 4.9 To See All The Commands You Have In Your Bot
+
+To see all the commands available in your bot, you may type in **/help** and the bot will display the commands available in the bot in the following manner:
+```
+Basic Commands 
+/start - Send welcome message 
+/help - list available commands 
+/quit - Send good bye message 
+
+General Commands 
+/isfree - To check whether you are free at a certain time interval 
+/getupcomingevent - List your upcoming events 
+
+Event-related Commands 
+/addevent - Add an event to your Google Calendar 
+/removeevent - Remove an event from your Google Calendar 
+/getevent - List all events that you have added 
+
+Course-related Commands 
+/addcourse - Add a course schedule to your Google Calendar 
+/removecourse - Remove a course schedule from your Google Calendar 
+/getcourse - List all courses that you have added 
+/setstudenttype - Set your student type (Full Time or Part Time) 
+/addfirstweek - Add the first weekday, i.e. Monday, of your first week and recess week
+```
+
+## Chapter 5: Further Information
+### Information 1: 
+
+For any commands that requires you to input a **date**:
+
+If the bot returns you this response:
+```
+Your format is correct, however we cannot perform the query to your Google Account
+```
+Then these cases may occur:
+
+Case 1: You have problems with your API keys
+
+Case 2: You entered a bad date, for instance your end time is earlier than your start time (Your event starts at 5 p.m. but ends at 4 p.m.)
+
+For these two cases, you may want to resolve your API problem and run the **/addevent** again and give the bot a reasonable time interval.
+
+### Information 2
+
+If the bot responds in this following manner:
+```
+Cannot access the course
+```
+This can also be caused by these following cases:
+
+Case 1: You may have problems with your browser driver, for instance you may have a problem with your chromedriver for Google Chrome.
+
+Case 2: You may have entered a course code that **does not exist**
+
+Case 3: You may have entered the wrong first week and first recess week. The **/addfirstweek** command can only accept a **Monday** as the start of the first week and first recess week.
```

Comment:
I don't think this is the case for **cannot access the course**, because the error 

> You may have entered the wrong first week and first recess week. The **/addfirstweek** command can only accept a **Monday** as the start of the first week and first recess week.

is an error that will always occur before user runs /addcourse, that is, because the users haven't set or falsely set their first weeks accordingly